sql/dd/impl/bootstrap/bootstrapper.cc

```diff
@@ -79,13 +79,37 @@ using namespace dd;
 
 namespace {
 
+// This method is required for upgrade sceanrio
+// during upgrade, there are two dd engine: actual engine and target engine
```

Comment:
```suggestion
// during upgrade, there are two dd engines: current engine and target engine
```

Reply:
Let me add both.. since in code(upstream), it also use actual..yeah, but 'current' maybe easy to understand.